rskj-core/src/main/java/co/rsk/peg/union/UnionBridgeSupportImpl.java

```diff
@@ -114,6 +114,53 @@ public Optional<Coin> getLockingCap() {
         return Optional.of(storageProvider.getLockingCap(activations).orElse(constants.getInitialLockingCap()));
     }
 
+    @Override
+    public int increaseLockingCap(Transaction tx, Coin newCap) {
+        final String INCREASE_LOCKING_CAP_TAG = "increaseLockingCap";
+
+        if (isUnauthorizedCaller(tx)) {
+            return UnionResponseCode.UNAUTHORIZED_CALLER.getCode();
+        }
+
+        if (isInvalidLockingCap(newCap)) {
+            return UnionResponseCode.INVALID_VALUE.getCode();
+        }
+
+        storageProvider.setLockingCap(newCap);
+        logger.info("[{}] Union Locking Cap has been increased. New value: {}",
+            INCREASE_LOCKING_CAP_TAG, newCap.value);
+        return UnionResponseCode.SUCCESS.getCode();
+    }
+
+    private boolean isInvalidLockingCap(Coin newCap) {
+        Coin currentLockingCap = storageProvider.getLockingCap(activations)
+            .orElse(constants.getInitialLockingCap());
+
+        if (newCap.compareTo(currentLockingCap) < 1) {
+            logger.warn(
+                "[{}] Attempted value doesn't increase Union Locking Cap. Value attempted: {} . currentLockingCap: {}",
+                "isInvalidLockingCap", newCap.value, currentLockingCap.value);
```

Comment:
Why `"isInvalidLockingCap"` passed as a parameter to the log and not included directly in the message? It's constant, not variable

Reply:
Updated